agent/functional_tests/util/utils_unix.go

```diff
@@ -38,6 +37,7 @@ import (
 	dockercontainer "github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/network"
 	docker "github.com/docker/docker/client"
+	"github.com/pkg/errors"
```

Comment:
IDE mistake?

Reply:
probably autogenerated when i changed something and removed it in this file.. will undo the change

Reply:
fixed